### ISSUE TITLE
bookmarks appear under sideberry panel until it fades in (only on new-tab)

### ISSUE THREAD

**Issue by marishlucas** [KiKaraage/ArcWTF#36]
https://github.com/KiKaraage/ArcWTF/assets/54138203/ad37a434-f2f4-467f-aed5-57efc2c1ffa6

**Comment by KiKaraage** [comment]
Yup that's because I had a bug when both sidebar and browser-container overlapped bookmarks toolbar when "show bookmarks toolbar in new tab only" - #30. Currently I have no solution yet...

**Comment by Enigma1309** [comment]
Disable bookmark bar altogether or add two flex space using firefox customization if you don't wan't bookmark
